test/kubetest/pods/proxynsmd.go

```diff
@@ -72,6 +72,16 @@ func ProxyNSMgrPodWithConfig(name string, node *v1.Node, config *NSMgrPodConfig)
 							ContainerPort: 5005,
 						},
 					},
+					Env: []v1.EnvVar{
+						{
+							Name: "NODE_NAME",
```

Comment:
Note: We already have this env for helm: https://github.com/networkservicemesh/networkservicemesh/blob/master/deployments/helm/proxy-nsmgr/templates/proxy-nsmgr.tpl#L38. So this patch should be work for helm cases.

Reply:
Nice.